egs/wham/DPRNN/train.py

```diff
@@ -66,30 +66,26 @@ def main(conf):
 
     # Define callbacks
     checkpoint_dir = os.path.join(exp_dir, 'checkpoints/')
-    checkpoint = ModelCheckpoint(checkpoint_dir, monitor='val_loss',
-                                 mode='min', save_top_k=5, verbose=1)
+    checkpoint = ModelCheckpoint(checkpoint_dir, monitor='val_loss', verbose=True,
+                                 save_last=False, save_top_k=5, save_weights_only=False, 
+                                 mode='min', period=1, prefix='')
```

Comment:
You don't need to write the default values if you don't change them. 
```suggestion
    checkpoint = ModelCheckpoint(checkpoint_dir, monitor='val_loss', verbose=True,
                                 save_top_k=5')
```

Reply:
I know. Do you prefer to delete the default values?

Reply:
Yes, I think it's clearer in this way.